FIPS/src/test/java/net/snowflake/client/jdbc/ConnectionFipsIT.java

```diff
@@ -290,6 +291,7 @@ public void testConnectUsingKeyPair() throws Exception {
   }
 
   @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGCP.class)
```

Comment:
for the conditional ignore let's add info why do we skip the tests on some environments

Reply:
@sfc-gh-dprzybysz  Done